vue/components/ui/atoms/loader/loader.vue

```diff
@@ -6,11 +6,10 @@
 
 <style lang="scss" scoped>
 @import "css/variables.scss";
+@import "~loaders.css";

```

Comment:
why this change here?

Reply:
I've described it in the `Decision`: having it in the `script` section requires adding a separate CSS loader for Jest, which doesn't make much sense.

Reply:
forget jest